root/etc/e-smith/db/configuration/migrate/Remove_logs_delegation

```diff
@@ -0,0 +1,18 @@
+{
+    #
+    # Remove the `logs` delegation to users  
+    #
+    use esmith::ConfigDB;
+    
+    my $db = esmith::ConfigDB->open();
+    my $newProp = '';
+    my @Keys = split(',', $db->get_prop('cockpit.socket','delegation')|| '');
+
+    foreach (@Keys) {
+        my @props = split(':',$_);
+        my @props = grep { !/logs/ } @props;
+        $newProp .= join(':',@props).',';
+    }
+    chop $newProp;
```

Comment:
Is it really safe?

Reply:
if we do not remove it, former delegation are still allowed

Reply:
done